### ISSUE TITLE
Add touch in addition to click

### ISSUE THREAD

**Issue by bgcart** [mhammond/pywin32#1464]
This is a feature request (also open to helping contribute) to add touch input to the pywin32 API.
It appears there is an equivalent input method to mouse_event, InjectTouchInput which would allow pywin32 to emulate touch. This would enable actions like auto-open of the on-screen keyboard on touch.

**Comment by tscizzlebg** [comment]
With the microsoft surface on Windows 10, this becomes a pretty natural extension. Would love to see this incorporated, and could be something I'd work on too.

**Comment by mhammond** [comment]
Please feel free to contribute!

**Comment by tscizzlebg** [comment]
Is there a guide for contributors? Like how to run tests, and such.

The README talks about building from source, but unclear to me if that's relevant or necessary or what, in order to develop.

My plan was to
1) clone the repo
2) get unit tests running/passing as a starting place before making any code changes
3) then try and follow the trail of how mouse_event works and copy that strategy for touch.

I cloned, then ran `python setup.py install`, and it told me I needed the Windows SDK, which I was about to go and get but then decided I'd post this question first so that it's up while I investigate on my own in parellel.

**Comment by mhammond** [comment]
The README is the only guide. Sadly you need the compiler and sdk to be able to build things, which is going to be necessary here.

**Comment by tscizzlebg** [comment]
Makes sense. However I do have Visual Studio and checked off the C++ workload, which I thought would supply the compiler and SDK.

Is there a step I'm overlooking? (before I'm able to run `python setup.py install`, and then I presume `python pywin32_testall.py`) Like to point to the compiler and SDK somehow?

**Comment by mhammond** [comment]
I suspect that `find_platform_sdk_dir()` simply doesn't know about the location of the SDKs installed by the later compilers. If you have a look at that function you will find it's possible to set a couple of env vars which might help, or if you don't mind shaving a yak or 2, you could try updating it to find it automatically.
